client/landing/jetpack-cloud/index.js

```diff
@@ -85,7 +85,7 @@ export default function() {
 
 		if ( config.isEnabled( 'jetpack-cloud/scan-history' ) ) {
 			page(
-				'/scan/:site/history',
+				'/scan/history/:site/',
```

Comment:
I like this change. Is there a reason behind it?

Reply:
the reason for it is to be more in line with how calypso works. It should probably be in a different PR. but it made hightlighting work might easier.

Reply:
Thanks for the explanation. It makes 100% sense!